src/lucky/static_compression_handler.cr

```diff
@@ -0,0 +1,87 @@
+# Middleware that serves static files that have been pre-compressed.
+# There can be multiple instances and the first in the middleware stack will take precedence.
+# For example, if you want to serve brotli compressed assets for browsers that support it and
+# serve gzip assets for those that don't you would do something like this in your middleware:
+#
+# ```
```

Comment:
May be worth putting a note that this is in `src/app_server.cr`

Reply:
Good call.